.github/workflows/board-automation.yaml

```diff
@@ -0,0 +1,20 @@
+name: Stripey
+
+on:
+  issues:
+    types:
+      - opened
+      - reopened
+permissions:
+    issues: write
```

Comment:
Hmm, if we need `repository-projects: write` to move issues into a project, does that mean we only need `issues: read`? (Since it doesn't _quite_ make sense to need write on both sides… not sure, though.)

Reply:
let's keep with the issues:write only... it makes more sense though.

Reply:
Can we get away with
```yaml
permissions:
    issues: read
    repository-projects: write
```

(The answer may be _no_; but it's useful to find out.)